types/node/index.d.ts

```diff
@@ -311,6 +316,22 @@ declare namespace NodeJS {
         new(stdout: WritableStream, stderr?: WritableStream): Console;
     }
 
+    export interface CallSite {
+        getThis: Function; // returns the value of this
```

Comment:
You could avoid the very generic type `Function` with a more concrete `getThis(): any` which clarifies that no arguments are expected.